### PR TITLE
Update qgrid for jupyterlab 

### PR DESCRIPTION
**Pull request by gmanoim-quantopian**
Credit to @qzchenwl via https://github.com/quantopian/qgrid/pull/262 for the work. 

- Bump dependencies (maintaining support for older jupyters/labs)
- bump version
- update css for lab

### CODE REVIEW

**Comment by wpence** [comment]
I have confirmed that this update works for both Jupyter Notebook 6.0.1 and Jupyter Lab 1.1.4. Would love to see this merged in.

File: js/src/qgrid.css [hunk 1]
```diff
@@ -737,3 +737,7 @@ input.bool-filter-radio {
 .q-grid .slick-sort-indicator-asc {
   background-image: none;
 }
+
+.jupyter-widgets {
+  overflow: auto !important;
+}
```
Comment (ssanderson) [inline js/src/qgrid.css]:
```suggestion
}
```

**Comment by people-can-fly** [comment]
when is the new version be available in pypi. pypi still has 1.1.1 released in Jul 11, 2018

**Comment by gerrymanoim** [comment]
> when is the new version be available in pypi. pypi still has 1.1.1 released in Jul 11, 2018

https://pypi.org/project/qgrid/1.2.0/

**Comment by people-can-fly** [comment]
Thanks for this new years gift. 😃

**Comment by tkrabel** [comment]
@gerrymanoim would you also do an `npm publish` of the JS extension so that people working with qgrid in JupyterLab get the correct extension without having to locally install from a github repo?

**Comment by gerrymanoim** [comment]
@tkrabel - Sorry for the delay, we had some account issues. To remedy that, we've pushed the package to a new home in https://github.com/quantopian/qgrid/pull/288. You will now need to `jupyter labextension install qgrid2`.

**Comment by ghuname** [comment]
I have just installed a fresh jupyterlab installation. I went to (GUI) extension manager and installed `@jupyter-widgets/jupyterlab-manager`, than I executed `jupyter labextension install qgrid2`, restarted jupyterlab server but I got an "`Error displaying widget: model not found`" error.

Looks like we still have issues here.

```
>jupyter labextension list
JupyterLab v2.0.0
Known labextensions:
   app dir: C:\Users\user\Miniconda3\share\jupyter\lab
        @jupyter-widgets/jupyterlab-manager v2.0.0 enabled  ok
        qgrid2 v1.1.2 enabled  ok
```

**Comment by aparente-nurix** [comment]
I'm having this exact same issues. Qgrid works fine in Jupyter notebook but not Jupyter lab. 


Error:

QgridWidget(grid_options={'fullWidthRows': True, 'syncColumnCellResize': True, 'forceFitColumns': False, 'defa…

Jupyterlab version: 
1.2.6

Qgrid version:
1.30 (installed via pip install qgrid --upgrade)

Pandas version: 1.03

Installed and built qgrid2-labextension based on the readme instructions. This is currently the only extension installed:

JupyterLab v1.2.6
Known labextensions:
   app dir: /anaconda3/share/jupyter/lab
        qgrid2 v1.1.2  enabled  OK

Uninstalled core extensions:
    @jupyter-widgets/jupyterlab-manager
    jupyterlab-chart-editor
    jupyterlab-plotly
    plotlywidget


Running on CentOS 7. Help would be greatly appreciated! I've exhausted every option in this thread, including downgrading Jupyter lab in a new Conda environment. Nothing has worked.

**Comment by da-phil** [comment]
> I have just installed a fresh jupyterlab installation. I went to (GUI) extension manager and installed `@jupyter-widgets/jupyterlab-manager`, than I executed `jupyter labextension install qgrid2`, restarted jupyterlab server but I got an "`Error displaying widget: model not found`" error.
> 
> Looks like we still have issues here.
> 
> ```
> >jupyter labextension list
> JupyterLab v2.0.0
> Known labextensions:
>    app dir: C:\Users\user\Miniconda3\share\jupyter\lab
>         @jupyter-widgets/jupyterlab-manager v2.0.0 enabled  ok
>         qgrid2 v1.1.2 enabled  ok
> ```

I have the same issue :(
```bash
> conda list | grep jupyter
jupyter                   1.0.0                      py_2    conda-forge
jupyter_client            6.1.3                      py_0    conda-forge
jupyter_console           6.1.0                      py_1    conda-forge
jupyter_contrib_core      0.3.3                      py_2    conda-forge
jupyter_contrib_nbextensions 0.5.1                    py37_0    conda-forge
jupyter_core              4.6.3            py37hc8dfbb8_1    conda-forge
jupyter_highlight_selected_word 0.2.0                 py37_1000    conda-forge
jupyter_latex_envs        1.4.6                 py37_1000    conda-forge
jupyter_nbextensions_configurator 0.4.1                    py37_0    conda-forge
jupyterlab                2.1.2                      py_0    conda-forge
jupyterlab_server         1.1.1                      py_0  
```

```bash
> jupyter labextension list         
JupyterLab v2.1.2
Known labextensions:
   app dir: /home/q477002/anaconda3/envs/machinelearning/share/jupyter/lab
        @jupyter-widgets/jupyterlab-manager v2.0.0  enabled  OK
        @jupyterlab/latex v2.0.0  enabled  OK
        jupyter-matplotlib v0.7.2  enabled  OK
        qgrid2 v1.1.3  enabled  OK
```

**Comment by robingirard** [comment]
Any idea on how to solve this issue ?

**Comment by mmlynarik** [comment]
I add to the people above, why this problem has still not been fixed? It's so sad that such useful tool is not usable because of technical problems.

**Comment by semihdesticioglu** [comment]
I wanted to try it on jupyterlab but I got error too.

**Comment by thomas-bc** [comment]
Same issue here. No updates in more than 2 years - is qgrid not maintained anymore?

**Comment by j-baum** [comment]
Same problem here, both on fresh anaconda 2022.05 and 2023.03 installs. Is there a fix?

EDIT: for anyone else landing here from a google search: this project is indeed no longer maintained. An alternative that has been recommended is https://github.com/bloomberg/ipydatagrid.

**Pull request merged**